doc/stream.tex

```diff
@@ -146,6 +156,26 @@
 (0 1 2 3 4)
 \codeend
 
+\defineentry{stream-lift}
+\begin{procedure}
+  \code{(stream-lift \var{p})}
+\end{procedure}
+\returns{} a procedure
+
+The \code{stream-lift} procedure returns a procedure that converts its argument with
+\code{unstream} then passes it to \var{p}. It can be used to include a non-stream-aware
+procedure in a stream pipeline.
+
+\defineentry{unstream}
+\begin{procedure}
+  \code{(unstream \var{x})}
+\end{procedure}
+\returns{} see below
+
+The \code{unstream} procedure converts \var{x} to a non-stream value. If \var{x} is a
```

Comment:
It might convert _x_ to a stream value.

The `unstream` procedure converts _x_ the same way `s/>` converts the final value it returns.

Reply:
I corrected unstream's description and changed `s/>` to reference it.